src/nft/types/navbar/navbar.ts

```diff
@@ -1,28 +1,48 @@
+import { NATIVE_CHAIN_ID } from 'constants/tokens'
+import { SafetyLevel } from 'graphql/data/__generated__/types-and-hooks'
+import { SearchToken } from 'graphql/data/SearchTokens'
+import { CHAIN_NAME_TO_CHAIN_ID } from 'graphql/data/util'
+
 export interface LooksRareRewardsData {
   address: string
   cumulativeLooksAmount: string
   cumulativeLooksProof: string[]
 }
-interface BridgeInfoEntry {
-  tokenAddress?: string
-}
-
-interface FungibleTokenExtensions {
-  bridgeInfo?: { [chain: number]: BridgeInfoEntry }
-}
 
 export interface FungibleToken {
   name: string
   address: string
   symbol: string
   decimals: number
   chainId: number
-  logoURI: string
-  coinGeckoId: string
-  priceUsd: number
-  price24hChange: number
-  volume24h: number
-  onDefaultList?: boolean
-  extensions?: FungibleTokenExtensions
-  marketCap: number
+  logoURI?: string | null
+  priceUsd?: number | null
+  price24hChange?: number | null
+  volume24h?: number | null
+  isVerified?: boolean
+}
+
+export function parseFungibleTokens(tokens: SearchToken[]) {
```

Comment:
this function should probably be part of `SearchToken.ts` and the useSearchTokens returns FungibleToken[].

Reply:
Both this function and the FungibleToken type will be deleted soon, see comment below